llarp/router/i_outbound_message_handler.hpp

```diff
@@ -0,0 +1,43 @@
+#ifndef LLARP_ROUTER_I_OUTBOUND_MESSAGE_HANDLER_HPP
```

Comment:
abstract instead?

Reply:
Yeah, abstract is fine.  Will change for all.

Reply:
See comment above, not changed.